sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -5,7 +5,6 @@
 /* eslint-disable @azure/azure-sdk/ts-apisurface-supportcancellation */
 
 // This file makes more sense if ordered based on how meaningful are some methods in relation to others.
```

Comment:
nit: I think this comment is no longer relevant - would you mind removing it as well? If I understand correctly it was there to justify the eslint-disable line

Reply:
Done